### ISSUE TITLE
bro please return funds i saved them for nearly 2 years please bro am begging you my wallet 

### ISSUE THREAD

**Issue by Shadow4friend** [percybolmer/DevToken#1]

**Comment by Shadow4friend** [comment]
0x525a867128DeaA9257C0b627e0dA97d22eBc1E90

**Comment by Shadow4friend** [comment]
i was very poor bro , i am collecting them for hospital treatment

**Comment by percybolmer** [comment]
May I ask what funds?

**Comment by Shadow4friend** [comment]
i used your code to learn it , when i pasted my memonic in . file all my funds were taken by someone ..

**Comment by Shadow4friend** [comment]
0x525a867128DeaA9257C0b627e0dA97d22eBc1E90 here is my address please have alook sir

**Comment by Shadow4friend** [comment]
https://bscscan.com/address/0x9cd02101b438f35072d53eaebe6c76be088e4ea8 this is the scammer scammer address

**Comment by tunnckoCore** [comment]
How much? lol.

and kill me, but i don't get what is happening :rofl: 

Relax and write a longer description

**Comment by percybolmer** [comment]
Yes, we would need more context here to figure out what is going on. 

If you simply inserted your memonic in then it should not do anyting unless you ran some code. 

We need more intel to help you on this matter.
